src/ACEtk/block/PhotoatomicSubshellTransitionData/src/ctor.hpp

```diff
@@ -0,0 +1,32 @@
+PhotoatomicSubshellTransitionData() = default;
+
+PhotoatomicSubshellTransitionData( const PhotoatomicSubshellTransitionData& ) = default;
+PhotoatomicSubshellTransitionData( PhotoatomicSubshellTransitionData&& ) = default;
+
+/**
+ *  @brief Constructor
+ *
+ *  @param[in] energies    the energy values
+ *  @param[in] loss        the average excitation energy loss values
```

Comment:
This constructor documentation is inconsistent with its contents. This might be fixed in a future PR (but really, this is why you don't do 10 PRs on one topic).

Reply:
That has been fixed in #112. By then this was changed to src/ACEtk/electron/SubshellTransitionData/src/ctor.hpp

Reply:
We all know why we try to do it this way ...